PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,28 +1,16 @@
-### Check list
+### Description
 
-> Check if done, if not relevant leave unchecked.
+_Short explanation of this PR (feel free to re-use commit message)_
```

Comment:
Yes, the "why" is really important. Sometimes I have hard time to understand the goal of a PR (what does it bring) :+1: